### DISCUSSION TITLE
Welcome to YAMMU Discussions!

### DISCUSSION THREAD

**Discussion by Zergie** [Zergie/YAMMU#21] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by akinferno** [comment]
There is no link to a community page or email or Discord channel, to discuss this project or volunteer to help. Where do I go to talk to you or any of your contributors about it?  My name is AKinferno on Discord as well.

**Reply by alexhayashi** [reply]
I second this, I was wondering if there was a discord to see progress updates

**Reply by Zergie** [reply]
Thanks for pointing this out — you’re absolutely right, and sorry for the missing links earlier 🙏

We’ve now set up a simple Discord server for YAMMU. It’s mainly there to give the project a place for discussion and to keep you updated on progress, rather than a large or complex community setup.

👉 Discord: https://discord.gg/QSjW5CMHZC

We’ll use it to share development updates, milestones, and occasional design discussions.
The README will be updated soon to include the link so it’s easier to find.

Thanks a lot for the interest — happy to have you along, and see you on Discord!
